docs/user-guide/configuration/index.md

```diff
@@ -18,7 +18,11 @@ You can access information about properties by hovering over the property name.
             - <span class="value">validate-west</span></a>
 <a href="#shared"><span class="key">shared</span>:</a>
     <a href="#environment"><span class="key">environment</span>:
-        <span class="key">NODE_ENV</span>: <span class="value">test</span></a>
+    <span class="key">NODE_ENV</span>: <span class="value">test</span></a>
+    <a href="#settings"><span class="key">settings</span>:</a>
+        <a href="#email"><span class="key">email</span>:
+    <span class="key">addresses</span>: <span class="value">[test@email.com, test2@email.com]</span>
```

Comment:
Which previewer are you using? Here's how it looks like when `mkdocs serve` is run.

<img width="575" alt="screen shot 2017-03-13 at 2 38 51 pm" src="https://cloud.githubusercontent.com/assets/2320956/23876470/e7ea88fa-07fa-11e7-8508-e3c1603dab79.png">